beacon-chain/rpc/eth/beacon/handlers_test.go

```diff
@@ -4754,3 +4755,213 @@ func Test_validateBlobSidecars(t *testing.T) {
 	require.NoError(t, err)
 	require.ErrorContains(t, "could not verify blob proof: can't verify opening proof", s.validateBlobSidecars(b, [][]byte{blob[:]}, [][]byte{proof[:]}))
 }
+
+func TestGetPendingDeposits(t *testing.T) {
+	config := params.BeaconConfig()
+	st, _ := util.DeterministicGenesisStateElectra(t, config.MaxValidatorsPerCommittee)
+
+	validators := st.Validators()
+	deps := make([]*eth.PendingDeposit, 20)
+	for i := 0; i < len(deps); i += 1 {
+		dummySig := make([]byte, 96)
+		for j := 0; j < 96; j++ {
+			dummySig[j] = byte(i)
+		}
+		deps[i] = &eth.PendingDeposit{
+			PublicKey:             validators[i].PublicKey,
+			WithdrawalCredentials: validators[i].WithdrawalCredentials,
+			Amount:                uint64(helpers.ActivationExitChurnLimit(1_000*1e9)) / 10,
```

Comment:
Is there any reason for using such a sophisticated formula instead of e.g. `123`? Someone reading this code later will wonder about it too.